src/test/regress/expected/rpt.out

```diff
@@ -1241,6 +1241,40 @@ select c from rep_tab where c in (select distinct d from rand_tab);
  2
 (2 rows)
 
+set optimizer= off;
```

Comment:
Please provide test description.
It's not recommended to limit test execution by some planner (except cases when it leads to creation new copy of out file, or reproduction steps uses fault injector and freezes on the different optimizer).

Reply:
done